indico/modules/events/registration/__init__.py

```diff
@@ -106,7 +107,9 @@ def _visible_registration(event):
                 .has_rows())
 
     def _visible_participant_list(event):
-        return event.has_feature('registration')
+        override = values_from_signal(signals.event.visible_participant_list.send(sender, event=event))
+        override = all(override) if override else None
+        return event.has_feature('registration') if override is None else override
```

Comment:
Should we even call the signal if the feature is disabled? I think we can return False early if the feature is disabled, and only then call the signal and take override into account..

Reply:
Okay, so the check can go something like 

```
        has_registration = event.has_feature('registration')
        if has_registration:
            override = values_from_signal(signals.event.visible_participant_list.send(sender, event=event))
            return all(override) if override else True
        return has_registration
```

Reply:
Much easier:

```
if not event.has_feature('registration'):
    return False
...
```

Reply:
```python
```